deploy/components/istio-control-plane/rbac.yaml

```diff
@@ -330,6 +330,21 @@ rules:
   verbs:
   - update
   - patch
+- apiGroups:
```

Comment:
ditto - remove from `inference.networking.x-k8s.io` group.

Reply:
The kgateway deployment files still use the v1alpha2 InferencePool